content/blog/creating-sigils.md

```diff
@@ -68,7 +69,7 @@ You can see the whole collection of our visual research [here](https://www.are.n
 
 Ultimately, we didn’t specifically draw on any of these influences very consciously. The goal of visual research is to get a sense of the territory, what has been tried, and what works. We think these threads of prior work are evident in the final result — you can *feel* them, but it’s tough to point to them individually.
 
-## What’s in a name?
+## What’s in a Name?
```

Comment:
I am pretty sure we use sentence casing. @collinscorrina ?

Reply:
Yes, you're right. @g-a-v-i-n can you update?

Reply:
you got it